examples/bilingual_document.ipynb

```diff
@@ -0,0 +1,305 @@
+      "----------------------------------------\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "Spanish: Estaba tumbado sobre su espalda dura, y en forma de caparazón y, al levantar un poco la cabeza, veía un vientre abombado, parduzco, dividido por partes duras en forma de arco, sobre cuya protuberancia apenas podía mantenerse el cobertor, a punto ya de resbalar al suelo.\n",
+      "----------------------------------------\n",
+      "Aligned Sentence Pairs (English -> German):\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "English: One morning, as Gregor Samsa was waking up from anxious dreams, he discovered that in bed he had been changed into a monstrous verminous bug.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Aligned Sentence Pairs (Spanish -> German):\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n",
+      "Spanish: Cuando Gregorio Samsa se despertó una mañana después de un sueño intranquilo, se encontró sobre su cama convertido en un monstruoso insecto.\n",
+      "German: Er lag auf seinem panzerartig harten Rücken und sah, wenn er den Kopf ein wenig hob, seinen gewölbten, braunen, von bogenförmigen Versteifungen geteilten Bauch, auf dessen Höhe sich die Bettdecke, zum gänzlichen Niedergleiten bereit, kaum noch erhalten konnte.\n",
+      "----------------------------------------\n"
+     ]
+    }
+   ],
+   "source": [
+    "from fastdtw import fastdtw\n",
```

Comment:
Usually, we put all the imports at the top of the file. 
I would adhere to the same style here.

Reply:
By the way, `fastdtw` is a nice solution!
Originally, I was thinking that you may want to implement the alignment algorithm yourself, but finding a tool that does exactly what you need is also totally right!